tests/PHPUnit/Framework/TestRequest/Response.php

```diff
@@ -262,4 +264,9 @@ private function removeSubtableIdsFromXml($apiResponse)
     {
         return $this->removeXmlFields($apiResponse, array('idsubdatatable_in_db'));
     }
+
+    private function removePiwikUrl($apiResponse)
+    {
+        return str_replace(Fixture::getRootUrl(), "", $apiResponse);
```

Comment:
would it maybe make sense to replace it with an absolute url such as `http://example.com/piwik`? In this case we would notice a difference between the root url is missing and the root url is not needed. I mean a test would still succeed if the domain was actually not present in the output. Not sure if it makes sense :)